CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ find_package(LLVM REQUIRED CONFIG)
 include_directories(${LLVM_INCLUDE_DIRS})
 separate_arguments(LLVM_DEFINITIONS_LIST NATIVE_COMMAND ${LLVM_DEFINITIONS})
 add_definitions(${LLVM_DEFINITIONS_LIST})
-llvm_map_components_to_libnames(llvm_libs codegen core support)
+llvm_map_components_to_libnames(llvm_libs codegen core support x86asmparser x86codegen x86desc x86disassembler x86info)
```

Comment:
I don't understand what adding x86asmparser x86codegen x86desc x86disassembler x86info is supposed to do, but will look into it and get back to you ~ APPROVED

Reply:
These are LLVM "components", parts of LLVM that can be used independently. Because we're all working on x86 processors, these libraries must be listed in order for the compiler to work. If we wanted to target other architectures, arm64 for example, we would add additional component names here.